### PR TITLE
bump build for rebuild // ensure license file is packaged

### PR DESCRIPTION
**Pull request by k-dominik**
Checklist
* [x] Used a fork of the feedstock to propose changes
* [x] Bumped the build number (if the version is unchanged)
* n/a Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/conda_smithy.html#how-to-re-render ) with the latest `conda-smithy`
* [x] Ensured the license file is being packaged.

in the current package version on conda-forge, the license file is not properly packaged (it is not in `info/license/`). I confirmed locally that a rebuild does the trick.

For reference, the extracted current package contents:

<details>

```
# termcolor-1.1.0-py_2.tar.bz2
├── info
│   ├── LICENSE.txt
│   ├── about.json
│   ├── files
│   ├── git
│   ├── hash_input.json
│   ├── index.json
│   ├── link.json
│   ├── paths.json
│   ├── recipe
│   │   ├── conda_build_config.yaml
│   │   ├── meta.yaml
│   │   └── meta.yaml.template
│   └── test
│       └── run_test.py
└── site-packages
    ├── termcolor-1.1.0-py2.7.egg-info
    │   ├── PKG-INFO
    │   ├── SOURCES.txt
    │   ├── dependency_links.txt
    │   ├── installed-files.txt
    │   └── top_level.txt
    └── termcolor.py
```

</details>

### CODE REVIEW

**Comment by k-dominik** [comment]
@conda-forge-admin, please rerender

**Comment by zklaus** [comment]
@conda-forge-admin, please rerender

**Review by zklaus — APPROVED** [review]

**Comment by zklaus** [comment]
This rebuild will also (hopefully) add proper metadata. Since the current build is over 4 years old, it only adds metadata to a Python 2.7 specific site-packages directory. @k-dominik, if you would like to be a maintainer, you might consider adding yourself to the maintainers list. @conda-forge/termcolor, could you have a look?

**Comment by k-dominik** [comment]
Hi @zklaus,

thanks for having a look! As for being a maintainer, termcolor doesn't seem like it's getting any new development, so it looks like minimal potential effort ;). I'll add myself.

**Comment by zklaus** [comment]
Cheers, @k-dominik, that is appreciated. After a change to the maintainers a rerender is necessary to update the README etc.

@conda-forge-admin, please rerender

File: recipe/meta.yaml [hunk 1]
```diff
@@ -11,16 +11,16 @@ source:
 
 build:
   noarch: python
-  number: 2
+  number: 3
   script: python -m pip install --no-deps --ignore-installed .
```
Comment (ocefpaf) [inline recipe/meta.yaml]: Let's take this opportunity and update this line to:

```suggestion
script: {{ PYTHON }} -m pip install . -vv
```

Comment (ocefpaf) [reply recipe/meta.yaml]: Can you also remove the line `fn: termcolor-{{ version }}.tar.gz` from above?

Comment (k-dominik) [reply recipe/meta.yaml]: sure, thanks!

**Comment by k-dominik** [comment]
I guess this could be merged @ocefpaf :)

**Comment by ocefpaf** [comment]
Thanks @k-dominik for modernizing our recipe and thanks @zklaus for the review!

**Pull request merged**